hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/message/MessageUtil.java

```diff
@@ -5,7 +5,10 @@
 
 public class MessageUtil {
 	
-	public static final Map<String, String> mTypeCollection = new HashMap<String, String>();
+	public static final Map<String, String> mTypeCollection = new HashMap<>();//NOSONAR 
```

Comment:
Whats this comment for?

Reply:
https://rules.sonarsource.com/java/RSPEC-1104

Added getter.